plugin-api/src/main/java/org/hyperledger/besu/plugin/data/BlockOverrides.java

```diff
@@ -392,13 +366,13 @@ public Builder extraData(final Bytes extraData) {
     }
 
     /**
-     * Sets the mix hash or previous Randao.
+     * Sets the mix hash
      *
-     * @param mixHashOrPrevRandao the mix hash or previous Randao to set
+     * @param mixHash the mix hash
      * @return the builder instance
      */
-    public Builder mixHashOrPrevRandao(final Hash mixHashOrPrevRandao) {
-      this.mixHashOrPrevRandao = mixHashOrPrevRandao;
+    public Builder mixHashOrPrevRandao(final Hash mixHash) {
+      this.mixHashOrPrevRandao = mixHash;
```

Comment:
should the method and field be renamed to remove "prevrandao"?

Reply:
renamed the field to mixHashOrPrevRandao (the call object accepts both mixHash & prevrandao)